src/ipc-handlers.ts

```diff
@@ -1235,7 +1238,11 @@ export function openCertificate( _event: IpcMainInvokeEvent ) {
 	return openCertificateDialog();
 }
 
-export function getFileContent( event: IpcMainInvokeEvent, filePath: string ) {
+export async function isCATrusted( _event: IpcMainInvokeEvent ): Promise< boolean > {
```

Comment:
```suggestion
export async function isCATrusted(): Promise< boolean > {
```
No need to include the `_event` argument anymore (after https://github.com/Automattic/studio/pull/1086 landed) if we don't use it.

Reply:
Thanks for the reminder. Removed it.